### DISCUSSION TITLE
Specify headers

### DISCUSSION THREAD

**Discussion by martinlundhgren** [Dedac/RestDataMappingPicklist#85] [Q&A]
Is it possible to specify headers in a request? The external API won't accept requests without a specific header set.

**Comment by Dedac** [comment]
It isn't currently, you can see in [rest-call](https://github.com/Dedac/RestDataMappingPicklist/blob/main/src/rest-call.ts)
that the parameters are hard coded to expecting a JSON result.
It wouldn't be a huge lift to add another input matching RestCallParameters named RestCallHeaders and do the same thing.  Would you like to do a PR?
